cmd/csi-adapter/cmd.go

```diff
@@ -46,9 +48,7 @@ var driverCmd = &cobra.Command{
 }
 
 func init() {
-	if Version == "" {
-		Version = "dev"
-	}
+	Version = "v0.0.1"
```

Comment:
is Version a flag or statically set?

Reply:
I would think it's statically set, I'm not sure where it would make sense for the user to set the version dynamically